ajax_with_jquery_exercise/nacker_hews_AJAX/style.css

```diff
@@ -0,0 +1,108 @@
+/*Block*/
+
+body {
+	font-size:18px;
```

Comment:
A small thing, but your indentation for the CSS is different than your indentation for the JS and HTML.